### DISCUSSION TITLE
MCDU text

### DISCUSSION THREAD

**Discussion by MStallen** [flybywiresim/aircraft#5055] [Help] [answered]
Is there a way to retreive the text on the MCDU by FSUIPC offsets or LVAR's?

**Comment by tracernz** [comment, marked as answer]
No. There's no way to write strings to SimVars at the moment. There is a request here that can be voted on: https://devsupport.flightsimulator.com/idea/264/support-strings-and-structs-in-lvars-and-allow-rea.html
